.github/workflows/tests.yml

```
@@ -80,11 +81,19 @@ jobs:
         id: changed
         run: |
           CHANGED="$(EXTRA=test .github/files/list-changed-projects.sh)"
+
+          # Only test certain plugins in combination with WC
+          if [[ "$WITH_WOOCOMMERCE" == true ]]; then
+            CHANGED=$( jq 'with_entries( select( .key == "plugins/jetpack" ) )' <<<"$CHANGED" )
```

This should fix the error:
```suggestion
            CHANGED=$( jq -c 'with_entries( select( .key == "plugins/jetpack" ) )' <<<"$CHANGED" )
```
(the problem was that the command we're using on line 91 can't handle if the variable has newlines)